base/applications/cmdutils/reg/lang/tr-TR.rc

```diff
@@ -1,102 +1,100 @@
-/* TRANSLATOR: 2015, 2016 Erdem Ersoy (eersoy93) (erdemersoy [at] erdemersoy [dot] net) */
```

Comment:
May need a file header; cc @binarymaster

Reply:
@HBelusca I have fixed the headers.